From 8fe31f02431b3680f09bf1346787276ddb2274a2 Mon Sep 17 00:00:00 2001
From: Andreas Enge <andreas.enge@inria.fr>
Date: Mon, 12 Dec 2022 15:30:57 +0100
Subject: [PATCH] Include header file.

This was detected by trying to compile with gmp-5.0.0 and mpfr-4.1.0.

* src/balls.c, src/eta.c: Include limits.h.
---
 src/balls.c | 1 +
 src/eta.c   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/balls.c b/src/balls.c
index 03a44a8..a18d37f 100644
--- a/src/balls.c
+++ b/src/balls.c
@@ -18,6 +18,7 @@ You should have received a copy of the GNU Lesser General Public License
 along with this program. If not, see http://www.gnu.org/licenses/ .
 */
 
+#include <limits.h> /* for CHAR_BIT */
 #include "mpc-impl.h"
 
 
diff --git a/src/eta.c b/src/eta.c
index 7d397f0..0651e15 100644
--- a/src/eta.c
+++ b/src/eta.c
@@ -19,6 +19,7 @@ along with this program. If not, see http://www.gnu.org/licenses/ .
 */
 
 #include <math.h>
+#include <limits.h> /* for CHAR_BIT */
 #include "mpc-impl.h"
 
 static void
-- 
GitLab

